clusters/ztp-overlays/skidoo/SiteConfig/kustomization.yaml

```
@@ -0,0 +1,46 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generators:
+ - skidoo-siteconfig.yaml
+
+resources:
+ - github.com/redhat-partner-solutions/vse-ocp-bases/base/ztp?ref=main
```

I think the `site-config` folder is missing here
```suggestion
 - github.com/redhat-partner-solutions/vse-ocp-bases/base/ztp/site-config?ref=main
```

---

thanks nati updated the change